app/views/content/is-teaching-right-for-me/engineers-teach-physics/_article.html.erb

```diff
@@ -63,7 +63,7 @@
      <p>You'll also be invited to an annual community day and have the opportunity to network with existing physics teachers.
      </p>
 
-    <p>You can <a href="https://www.find-postgraduate-teacher-training.service.gov.uk/">search for physics teacher training courses</a> and then filter by 'Engineers teach physics'.
+    <p><a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?engineers_teach_physics=true&degree_required=two_two&applications_open=false&l=2&subjects%5B%5D=F3">Find engineers teach physics teacher training courses</a>.
```

Comment:
Only suggestion, and happy to be overruled on it is to add the filter for 'courses that are open', which would be this URL:
`https://www.find-postgraduate-teacher-training.service.gov.uk/results?engineers_teach_physics=true&study_type%5B%5D=full_time&study_type%5B%5D=part_time&qualification%5B%5D=qts&qualification%5B%5D=pgce_with_qts&qualification%5B%5D=pgce+pgde&degree_required=two_two&applications_open=false&applications_open=true&l=2&subjects%5B%5D=F3`

It uses this filter: 
<img width="263" alt="image" src="https://github.com/DFE-Digital/get-into-teaching-app/assets/35870975/ebefab62-bb9b-475b-a34e-2464956b51a8">


This gives 16 results instead of the current 17. Is our aim to give users access to the full suite of courses, or to show them those that they can apply to at this specific moment? cc @gemmadallmandfe